docs/index.md

```diff
@@ -7,17 +7,35 @@ description: |-
 
 # Civo Provider
 
-The Civo provider is used to interact with the resources supported by Civo. The provider needs to be configured with the proper credentials before it can be used.
+The Civo provider is used to interact with the resources supported by Civo. 
 
 Use the navigation to the left to read about the available resources.
 
+## Configuration
+
+The provider will use the credentials of the [Civo CLI](https://github.com/civo/cli) (stored in ` ~/.civo.json`) if no other credentials have been set up. The credentials preference is the following:
+
+1. Environment variable (`CIVO_TOKEN`).
+1. Token provided via a credentials file (See `credentials_file` input [below](#credentials_file))
+1. [CLI](https://github.com/civo/cli) configuration (`~/.civo.json`)
```

Comment:
numbering is incorrect

Reply:
That's deliberate. Most markdown interpreters will automatically number them correctly, the reason why so many people do all 1s by convention is that if you need to add something in the middle you don't need to change the entire list.